yarn-project/aztec-nr/aztec/src/note/note_viewer_options.nr

```diff
@@ -42,4 +44,9 @@ impl<Note, N> NoteViewerOptions<Note, N> {
         self.offset = offset;
         *self
     }
+
+    pub fn include_nullified(&mut self) -> Self {
```

Comment:
Should this be a setter similar to the others instead? Allowing both on and off switching?